lib/init/features/database.js

```diff
@@ -1,17 +1,41 @@
 'use strict';
 
 var chalk = require('chalk');
-
+var api = require('../../api');
 var prompt = require('../../prompt');
 var logger = require('../../logger');
+var configstore = require('../../configstore');
+var utils = require('../../utils');
+var fsutils = require('../../fsutils');
+var RSVP = require('rsvp');
+
+var _getConsoleDBRules = function(projectId) {
+  return api.request('GET', '/.settings/rules.json', {
+    auth: true,
+    origin: utils.addSubdomain(api.realtimeOrigin, projectId)
+  }).then(function(response) {
+    return JSON.stringify(response.body, null, 2);
+  });
+};
+
+var _writeDBRules = function(projectId, fileName, config) {
+  return _getConsoleDBRules(projectId).then(function(rules) {
+    return config.writeProjectFile(fileName, rules);
+  }).then(function() {
+    utils.logSuccess('Database Rules for ' + projectId + ' have been downloaded to ' + fileName + ' .');
```

Comment:
Typically I'll `chalk.bold()` identifiers that are being called out in a log line.